sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -211,6 +211,52 @@ object JdbcUtils extends Logging {
   }
 
   /**
+   * Returns the schema if the table already exists in the JDBC database.
+   */
+  def getSchema(conn: Connection, url: String, table: String): Option[StructType] = {
+    val dialect = JdbcDialects.get(url)
+
+    Try {
+      val statement = conn.prepareStatement(dialect.getSchemaQuery(table))
+      try {
+        getSchema(statement.executeQuery(), dialect)
+      } finally {
+        statement.close()
+      }
+    } match {
+      case Success(v) =>
+        Some(v)
+      case Failure(e) =>
+        None
+    }
+  }
+
+  /**
+   * Returns the saving schema using rddSchema's sequence and tableSchema's name.
```

Comment:
`saving schema` is not right. We need a better name here. 

`rddSchema's sequence and tableSchema's name` -> `rddSchema's column sequence and tableSchema's column names`